### ISSUE TITLE
Unable to Focus on Charts View Using External Keyboard on iOS

### ISSUE THREAD

**Issue by lingzlu** [danielgindi/Charts#5095]
We have customers who rely solely on keyboard navigation. Our app permits interaction with the legend and bars on the BarChartView via touch. However, these UI elements are not accessible via keyboard but can be accessed when VoiceOver is enabled. The same issue is present in the demo app as well. Kindly advise if there's a step I might have overlooked to enable keyboard support, or if the library itself lacks keyboard accessibility. Thank you.

**Comment by msftedad** [comment]
Is any update on this issue ?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
